### PR TITLE
Node create file every second

### PR DESCRIPTION
**Pull request by vsupruniuk**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -1 +1,25 @@
...
+  const hours = date.getHours();
+  const minutes = date.getMinutes();
+  const seconds = date.getSeconds();
+
+  fs.writeFile(`src/app-${hours}_${minutes}_${seconds}.log`,
+    `${date}`,
+    (error) => {
+      if (error) {
+        console.log(error);
+      } else {
```
Comment (Olexandr6) [inline src/app.js:17]: Let's avoid `else` statement. It's redundant here

File: src/app.js [hunk 2]
```diff
@@ -1 +1,25 @@
...
+        console.log(`Created file name:
+      app-${hours}_${minutes}_${seconds}.log\n
```
Comment (Olexandr6) [inline src/app.js:19]: Let's create a variable for the file name, you used it twice, so it makes sense

File: src/app.js [hunk 3]
```diff
@@ -1 +1,25 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+
+const createFile = () => {
+  const date = new Date();
+  const hours = date.getHours();
+  const minutes = date.getMinutes();
+  const seconds = date.getSeconds();
```
Comment (Olexandr6) [inline src/app.js:10]: The time format should be `17:05:25` as the standard one. But in your case, it's: `17:5:25`.
If there is a digit < `10` , there should be the `0` before.

File: src/app.js [hunk 4]
```diff
@@ -1 +1,25 @@
...
+
+  fs.writeFile(`src/app-${hours}_${minutes}_${seconds}.log`,
+    `${date}`,
+    (error) => {
+      if (error) {
+        console.log(error);
```
Comment (Olexandr6) [inline src/app.js:16]: Let's throw an error instead. Your app shouldn't continue when there is an error

**Review by G-YaSha — CHANGES_REQUESTED** [review]
Gj👍 Let's work on improvemets

**Pull request closed without merging**